docs/data/xrootd/install-storage-element.md

```diff
@@ -273,27 +273,7 @@ fermicloud054.fnal.gov complete inventory as of Tue Apr 12 07:38:29 2011 /data/x
 
 XRootD can be accessed using the HTTP protocol. To do that:
 
```

Comment:
You need a step here saying that they should `set EnableHttp = 1` in `/etc/xrootd/config.d/10-local.cfg`

Reply:
Do you mean that this belongs in the `xrootd/install-standalone.md ` doc or the `xrootd/install-storage-element.md ` ?

Reply:
In `xrootd/install-storage-element.md` (along with a change to the install command from `yum install xrootd` -> `yum install osg-xrootd` at the top of the document

Reply:
Alright I've updated the yum install line and added instructions for the `set EnableHttp = 1` bit.

Although I think I left this out originally because `set EnableHttp = 1` is already provided by the package in [40-osg-standalone.cfg](https://vdt.cs.wisc.edu/svn/native/redhat/trunk/osg-xrootd/osg/40-osg-standalone.cfg).

Or did you mean earlier when you said

>Please remove the HTTP optional configuration section, that's built into the `osg-xrootd-standalone` metapackage now

that we want to remove these two lines from the `40-osg-standalone.cfg` that we ship?
```
set EnableHttp = 1
set EnableLcmaps = 1
```

Reply:
I'm saying that since we have those lines in `40-osg-standalone.cfg`, we can get rid of [this section](https://github.com/opensciencegrid/docs/blame/150ca9b97d666fb64dcac9c795acfcc83a5ba320/docs/data/xrootd/install-standalone.md#L90-L94) entirely. The user automatically gets HTTP/S support by installing `osg-xrootd-standalone`.

OTOH, the storage element document needs to be more explicit since we don't have a metapackage for the XRootD SE (and don't have plans to release one), so we have to tell users to add the following to their config to enable HTTP/S:

```
set EnableHttp = 1
set EnableLcmaps = 1
```

The current document only has the [EnableLcmaps line](https://github.com/opensciencegrid/docs/blame/150ca9b97d666fb64dcac9c795acfcc83a5ba320/docs/data/xrootd/install-storage-element.md#L280-L281).

Reply:
I see, sorry for the confusion.  Lol, the comma here (rather than a semicolon) really threw me:

> Please remove the HTTP optional configuration section, that's built into the `osg-xrootd-standalone metapackage` now

Anyway I've dropped the HTTP config section, and I added the instructions for adding the `set EnableHttp = 1` line to the `install-storage-element.md` doc.  However, I said in the instructions to put it in `10-common-site-local.cfg`, rather than `10-xrootd-lcmaps.cfg` (where the `set EnableLcmaps = 1` line appears).  But if you think the EnableHttp line belongs better in `10-xrootd-lcmaps.cfg` I can change that.

Reply:
Yeah, we should probably shove it into `10-xrootd-lcmaps.cfg` instead

Reply:
Alright.  It had seemed like maybe the EnableHttp line (which did not have a commented-out example in the xrootd-lcmaps config) might have belonged somewhere else, but maybe that idea was just unnecessarily complicated.  Anyway, updated.